src/components/MediaSettings/MediaSettings.vue

```diff
@@ -170,7 +170,7 @@
 					class="call-button"
 					:force-join-call="true"
 					:silent-call="silentCall" />
-				<NcButton v-else @click="closeModal">
+				<NcButton v-else @click="closeModalAndConfirmBackground">
 					{{ t('spreed', 'Done') }}
```

Comment:
```suggestion
				<NcButton v-else-if="updatedBackground" @click="closeModalAndConfirmBackground">
					{{ t('spreed', 'Apply changes') }}
```

Don't know why we had that button before, as modal could be closed with outside click, or with a cross 
cc @marcoambrosini

Reply:
The `mediaSettings `also has Devices settings. It will be weird to show `'Apply changes'` button only for the background change and not for changing the media devices. I'd prefer to keep it as a permanent `'Done'` button for any changes in general OR  show it for both changes. 

P.S: If we go with the conditioned button, I think it should be removed if the chosen option is the same as the current one in order to reflect 'real changes' only.